test/jdk/java/awt/List/ListEnterExitTest.java

```diff
@@ -93,38 +109,17 @@ public void start() throws Exception {
                 }
             });
         }
-        System.out.println("mouseEnterExitListener.isPassed() : " + mouseEnterExitListener.isPassed());
-        if (!mouseEnterExitListener.isPassed()) {
-            throw new RuntimeException("Haven't receive mouse enter/exit events");
-        }
-
     }
 
-}
-
-class MouseEnterExitListener extends MouseAdapter {
-
-    volatile boolean passed_1 = false;
-    volatile boolean passed_2 = false;
-
-    public void mouseEntered(MouseEvent e) {
-        passed_1 = true;
-        System.out.println("passed_1 is: " + passed_1);
-    }
-
-    public void mouseExited(MouseEvent e) {
-        passed_2 = true;
-        System.out.println("passed_2 is: " + passed_2);
-    }
-
-    public void mousePressed(MouseEvent e) {
-        synchronized (this) {
-            System.out.println("mouse pressed");
-            this.notifyAll();
+    static class MouseEnterExitListener extends MouseAdapter {
```

Comment:
```suggestion
    private class MouseEnterExitListener extends MouseAdapter {
```
Removing `static` allows removing `static` modifier from the latches.

In fact, the `MouseEnterExitListener` class can be eliminated altogether… Yet the code could be clearer this way.

Reply:
I thought of removing the class but then found it ok to have.